libs/auth/react/src/lib/userManager.ts

```diff
@@ -18,9 +24,12 @@ export const getAuthSettings = (): AuthSettings => {
   return authSettings
 }
 
-export const configure = (settings: AuthSettings) => {
-  authSettings = mergeAuthSettings(settings)
+export const configure = (settings: AuthSettingsWithArrayOfScope) => {
+  const scope = toStringScope(settings.scope)
+
+  authSettings = mergeAuthSettings({ ...settings, scope })
```

Comment:
Not a big fan of `AuthSettingsWithArrayOfScope`. 

Would it work to modify AuthSettings to have a scope array, then just pass it to UserManager like this:

```
userManager = new UserManager({
  ...authSettings,
  scope: toStringScope(authSettings.scope),
})
```

Either that, or have the AuthSettings interface (with string array) separate from the UserManager argument, eg:

```
userManager = new UserManager(toUserManagerOptions(authSettings))
```

Reply:
I went for your first solution 👍